nearlib/test-tools/in_memory_key_store.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Simple in-memory keystore for testing purposes.
+ */
+var keys = {};
+
+exports.setKey = (account_id, key) => {
```

Comment:
1) any API which can potentially do I/O (or heavy CPU work that can be off-loaded to bg thread) should either be `async` or return promises explicitly.
2) use object literal here as well?